app/src/main/AndroidManifest.xml

```diff
@@ -173,15 +167,17 @@
             </intent-filter>
             <intent-filter android:label="@string/label_share_intent"> <!-- Share action -->
                 <action android:name="android.intent.action.SEND" />
+
                 <category android:name="android.intent.category.DEFAULT" />
+
                 <data android:mimeType="text/plain" />
             </intent-filter>
         </activity>
         <activity
-            android:name=".MainActivity"
+            android:name="is.xyz.browse.BrowseActivity"
```

Comment:
everything should still be under `is.xyz.mpv`

Reply:
splitting to directory makes it easier to work with.
So should i maybe move it to, ``is.xyz.mpv.BrowseActivity``?